kitsune/sumo/static/sumo/js/form-wizard-sign-in-step.js

```diff
@@ -38,14 +33,14 @@ export class SignInStep extends BaseFormStep {
             <input name="redirect_immediately" value="" type="hidden"/>
             <input name="redirect_to" value="" type="hidden"/>
 
-            <label for="email">${gettext("Email")}</label>
+            <label for="email">${gettext("Enter your email")}</label>
             <div class="tooltip-container">
               <aside id="email-error" class="tooltip">${gettext("Valid email required")}</aside>
-              <input id="email" name="email" type="email" required="true" placeholder="${gettext("Enter your email address")}"/>
+              <input id="email" name="email" type="email" required="true" placeholder="${gettext("user@example.com")}"/>
             </div>
 
-            <button class="for-sign-up mzp-c-button mzp-t-product" type="submit" data-event-category="device-migration-wizard" data-event-action="click" data-event-label="signup-button">${gettext("Sign up")}</button>
-            <button class="for-sign-in mzp-c-button mzp-t-product" type="submit" data-event-category="device-migration-wizard" data-event-action="click" data-event-label="sign-in-button">${gettext("Sign in")}</button>
+            <button class="for-sign-up mzp-c-button mzp-t-product" type="submit" data-event-category="device-migration-wizard" data-event-action="click" data-event-label="signup-button">${gettext("Continue")}</button>
+            <button class="for-sign-in mzp-c-button mzp-t-product" type="submit" data-event-category="device-migration-wizard" data-event-action="click" data-event-label="sign-in-button">${gettext("Continue")}</button>
```

Comment:
Since we're using the same label now I wonder if we could simplify/clean things up a bit by just changing the `data-label` via JS.

Reply:
Great suggestion, will do, thanks!